tests/console/ping.pm

```diff
@@ -61,8 +65,19 @@ sub run {
         if (defined($bug)) {
             record_info('Softfail', $bug, result => 'softfail');
         } else {
+            record_info("Unknown failure on $cmd, maybe related to: bsc#1200617, bsc#1195826, bsc#1196840, bsc#1199918, bsc#1199926, bsc#1199927",
+                result => 'fail');
+            $self->result("fail");
+        }
+    }
+
+    if ($capability && $ping_group_range !~ m/^net.ipv4.ping_group_range\s*=\s*1\s*0/) {
+        my $msg = "capability '$capability' is not needed when ICMP socket allowed for non-root user: '$ping_group_range'";
+        if (is_sle('=15-SP3')) {
+            record_info('unneeded capability', "bsc#1196840#c29: $msg", result => 'softfail');
+        } else {
+            record_info('unneeded capability', $msg, result => 'fail');
```

Comment:
Isn't `result` here and on line below doing the same ?

Reply:
Well, softfail should have yellow border, fail red border. Or am I wrong and should prove it with a test run?

Reply:
What soft fail ? I mean `$self->result("fail");` should be not necessary with `result => 'fail'` in record_info or ?

Reply:
IMHO it was needed, but I'll recheck.

Reply:
@dzedro it is needed: http://quasar/tests/1596#step/ping/9
It does just read border around the box, but I want whole test to fail.

Reply:
Then sorry, it's just confusing.
What's the point of having red square. :)